presto-hive/src/main/java/com/facebook/presto/twitter/elephantbird/serde/ThriftSerDe.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twitter.elephantbird.hive.serde;
```

Comment:
Well we should move this class to it's proper home (without elephantbird) before we ship, since it's basically naming technical debt, which is much easier to fix before we go live than after.